### ISSUE TITLE
fails if cargo edition feature is used

### ISSUE THREAD

**Issue by matthiaskrgr** [RazrFalcon/cargo-bloat#24]
For example, running ```` cargo bloat````on  https://github.com/rust-lang-nursery/rust-clippy/ fails:
````
error: failed to parse manifest at `/home/matthias/vcs/github/rust-clippy/Cargo.toml`
Caused by:
  unknown cargo feature `edition`
````

**Comment by RazrFalcon** [comment]
Yes. `cargo` should be updated.

**Comment by RazrFalcon** [comment]
Fixed in 66a9291cd891f5a125a8ac5247f3ac0b999177a4
